src/tests/ftest/datamover/dm_obj_large_posix.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python
+'''
+  (C) Copyright 2020-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+'''
+from data_mover_test_base import DataMoverTestBase
+
+
+# pylint: disable=too-many-ancestors
+class DmObjLargePosix(DataMoverTestBase):
+    """Test class Description:
+       Verify cloning a large POSIX container at the object level (non-DFS)
+       using dcp.
+
+    :avocado: recursive
+    """
+
+    def run_dm_obj_large_posix(self, tool):
```

Comment:
It would be great if you could add description for `tool`.

Reply:
Added!